test/support/active_record.rb

```diff
@@ -74,9 +74,11 @@ def deserialize(value)
 
   store :credentials, accessors: [:username, :password_ciphertext], coder: JSON
   store :credentials2, accessors: [:username2, :password2_ciphertext], coder: JSON
+  store :credentials3, accessors: [:username3, :password3_ciphertext], coder: JSON
```

Comment:
the original add here was this...

```
store :credentials3, accessors: [:username3], coder: JSON
```

I changed it to add the `password3_ciphertext` inside the accessors array, as I noticed that was from one of our changes.